apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryGeography.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright 2019 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+
+namespace Google.Cloud.BigQuery.V2
+{
+    /// <summary>
+    /// Representation of the BigQuery GEOGRAPHY type, representing geospatial information.
+    /// Equality is determined on a simple textual basis.
+    /// </summary>
+    public sealed class BigQueryGeography : IEquatable<BigQueryGeography>
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public string Text { get; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="text"></param>
+        private BigQueryGeography(string text)
+        {
+            Text = GaxPreconditions.CheckNotNull(text, nameof(text));
```

Comment:
nit: consider using an expr-bodied ctor?

Reply:
Yup, might as well for now.